accessibility-checker-engine/src/v2/checker/accessibility/nls/index.ts

```diff
@@ -886,15 +886,15 @@ let a11yNls = {
     },
     // JCH - DONE
     "IBMA_Focus_Tabbable": {
-        0: "Component must have at least one tabbable element",
+        0: "Component must have at least one tabable element",
```

Comment:
Tabbable was correct

Reply:
agree. These one shoud not have been changed. correcting back